confetti.js

```diff
@@ -26,14 +31,14 @@ const bang = (addConfettiParticles) => {
   // sprider ut sig från varandra samt ett gruppnamn som skickas in i
   // chooseParticleColor. Ändra på dessa värden och se vad som händer!
   addConfettiParticles({
-    groupName: 'lonely sad confetti',  // Valfritt namn på gruppen
-    particleAmount: 1,  // Antal konfettipartiklar
-    xPosition: 50,      // Position i procent av skärmens bredd, 0 är längst till vänster och 100 längst till höger
-    yPosition: 50,      // Position i procent av skärmens höjd, 0 är längst ner och 100 högst upp
-    xVelocity: 0,       // Hastighet i sidled, positiva värden åker åt höger och negativa åt vänster
-    yVelocity: 2,       // Hastighet i höjdled, positiva värden åker uppåt och negativa värden nedåt
-    xSpread: 80,        // Slumpmässig spridning på partikelns hastighet i sidled
-    ySpread: 20         // Slumpmässig spridning på partikelns hastighet i höjdled
+    groupName: 'happy colorful confetti',  // Valfritt namn på gruppen
+    particleAmount: 1800,  // Antal konfettipartiklar
+    xPosition: 100,      // Position i procent av skärmens bredd, 0 är längst till vänster och 100 längst till höger
+    yPosition: 100,      // Position i procent av skärmens höjd, 0 är längst ner och 100 högst upp
+    xVelocity: 2,       // Hastighet i sidled, positiva värden åker åt höger och negativa åt vänster
+    yVelocity: 1,       // Hastighet i höjdled, positiva värden åker uppåt och negativa värden nedåt
+    xSpread: 90,        // Slumpmässig spridning på partikelns hastighet i sidled
+    ySpread: 90        // Slumpmässig spridning på partikelns hastighet i höjdled
   })
```

Comment:
Snyggt att du har ändrat på konfetti så att den kommer ner från toppen istället från at skjutas upp från knappen, det passade bra till ditt tema med födelsedagskort. 

Inte så mycket mer att påpeka med den, jag tipsar häller för inspiration om man vill leka mer:
Ett tips för inspiration är att testa göra ett konfettiregn som repeterar för alltid när man klickar på knappen, och som stoppar igen när man klickar på knappen en gång till.
Då kan man välja använda setInterval som fungerar som setTimeout fast den kommer repetera för alltid tills man stoppar den igen. Den tar in en funktion att repetera och returnerar ett ID som man kan spara och använda när man vill stoppa den.

`let intervalId = setInterval(addConfettiParticles( ... ), 500);`
Då kan man stoppa den igen med:
`clearInterval(intervalId)`

ett exempel på det:
```
let intervalId;
const bang = (addConfettiParticles) => {
  if (intervalId) {
    clearInterval(intervalId);
    intervalId = undefined;
  } else {
    intervalId = setInterval(addConfettiParticles({ ... }), 1000);
  }
};